### PR TITLE
HybridDataManager - Adding SDK version 1.0.1 to correspond to API version 2019-06-01

### PR DESCRIPTION
**Pull request by prtank**

### CODE REVIEW

**Comment by prtank** [comment]
PR for spec merge of API version 2019-06-01 is https://github.com/Azure/azure-rest-api-specs/pull/8676

File: sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Microsoft.Azure.Management.HybridData.csproj [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
     <PackageId>Microsoft.Azure.Management.HybridData</PackageId>
     <Description></Description>
-    <Version>1.0.0</Version>
+    <Version>1.0.1</Version>
     <AssemblyName>Microsoft.Azure.Management.HybridData</AssemblyName>
     <PackageTags>Management;Storage;StorSimple;StorSimpleDataManager;Azure;HybridDataManager;Microsoft.HybridData</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
       This is the first public release of the Azure StorSimple Data Manager SDK (Microsoft.HybridData).
+	  API version 1.0.1: Adding PartiallySucceeded job status
```
Comment (isra-fel) [inline sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Microsoft.Azure.Management.HybridData.csproj]:
```suggestion
Adding PartiallySucceeded job status
```
Please remove release notes from last release.

File: sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs [hunk 2]
```diff
@@ -361,7 +357,7 @@ private void Initialize()
             DataStoreTypes = new DataStoreTypesOperations(this);
             PublicKeys = new PublicKeysOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2016-06-01";
+            ApiVersion = "2019-06-01";
```
Comment (isra-fel) [inline sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs]: As your are changing the API version, please also update the api tags in this file: `AzSdk.RP.props`

Comment (isra-fel) [reply sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs]: By the way, does this API version update bring any breaking changes? If so, SDK will need a major version update.

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @prtank generally the API spec needs to be merged into master BEFORE you generate SDK and submit this PR.

I noticed that the PR is missing a change to `/eng/mgmt/mgmtmetadata/hybriddatamanager_resource-manager.txt`, also the license headers are difference. I suspect it's caused by running `autorest` directly, rather that our recommended way: running `src/generate.ps1` in your module.

So, please do the following things after the swagger review PR is merged:
1. Edit `src/generate.ps1`, change -AutoRestVersion "latest" to "2.0.4413"
1. Run it to generate SDK
    1. In case you encounter any error, please refer to our document: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk
1. Make sure you check in the txt file mentioned above

Thank you

**Comment by isra-fel** [comment]
kindly ping @prtank

**Comment by prtank** [comment]
Hi, 

I have been waiting for the swagger spec PR to be merged before I take care of these changes. I am expecting that to go through today so I will take care of all these comments before Monday.

Thanks for reviewing in advance :)

File: sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs (hunk 2 above)
Comment (prtank) [reply sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs:360]: The AzSdk.RP.props file is also changed in this PR

Comment (prtank) [reply sdk/hybriddatamanager/Microsoft.Azure.Management.HybridDataManager/src/Generated/HybridDataManagementClient.cs:364]: There is no breaking change in this API version upgrade. We are adding a new value to an enum and have updated some comments about the data models/APIs in use.

**Comment by prtank** [comment]
> Hi @prtank generally the API spec needs to be merged into master BEFORE you generate SDK and submit this PR.
> 
> I noticed that the PR is missing a change to `/eng/mgmt/mgmtmetadata/hybriddatamanager_resource-manager.txt`, also the license headers are difference. I suspect it's caused by running `autorest` directly, rather that our recommended way: running `src/generate.ps1` in your module.
> 
> So, please do the following things after the swagger review PR is merged:
> 
> 1. Edit `src/generate.ps1`, change -AutoRestVersion "latest" to "2.0.4413"
> 2. Run it to generate SDK
>    
>    1. In case you encounter any error, please refer to our document: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk
> 3. Make sure you check in the txt file mentioned above
> 
> Thank you

I have pushed the changed for AutoRest version 2.0.4413, but am curious as to why we are using that specific version instead of the latest - 3.0.6187

**Review by isra-fel — APPROVED** [review]
LGTM!

**Pull request merged**